clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -378,6 +380,7 @@ private PushJobSetting getPushJobSetting(VeniceProperties props) {
     pushJobSettingToReturn.batchNumBytes = props.getInt(BATCH_NUM_BYTES_PROP, DEFAULT_BATCH_BYTES_SIZE);
     pushJobSettingToReturn.isIncrementalPush = props.getBoolean(INCREMENTAL_PUSH, false);
     pushJobSettingToReturn.isDuplicateKeyAllowed = props.getBoolean(ALLOW_DUPLICATE_KEY, false);
+    pushJobSettingToReturn.largeRecordsAllowed = props.getBoolean(LARGE_RECORDS_ALLOWED, true);
```

Comment:
Can't we infer this from the store config (`!= -1`)? Why do we need an explicit config?

Reply:
[See this comment.](https://github.com/linkedin/venice/pull/1052#discussion_r1669462089)